src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -102,6 +103,21 @@
            (create-field name (display-value value)))
          log-map)])))
 
+(defn- failure-tree [failures]
+  [comps/Tree {:start-collapsed? false
+               :label "Failures"
+               :highlight-ends false
+               :data (prewalk
```

Comment:
prewalk: apply a function to the object and every sub-element of the object, transforming from the outside in.